mmtrack/models/mot/strong_sort.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, Optional
+
+from torch import Tensor
+
+from mmtrack.registry import MODELS, TASK_UTILS
+from mmtrack.utils import OptConfigType, SampleList
+from .deep_sort import DeepSORT
+
+
+@MODELS.register_module()
+class StrongSORT(DeepSORT):
+    """StrongSORT: Make DeepSORT Great Again.
+
+    Details can be found at `StrongSORT<https://arxiv.org/abs/2202.13514>`_.
+
+    Args:
+        detector (dict): Configuration of detector. Defaults to None.
+        reid (dict): Configuration of reid. Defaults to None
+        tracker (dict): Configuration of tracker. Defaults to None.
+        kalman (dict): Configuration of Kalman filter. Defaults to None.
+        cmc (dict): Configuration of camera model compensation.
+            Defaults to None.
+        data_preprocessor (dict or ConfigDict, optional): The pre-process
+           config of :class:`TrackDataPreprocessor`.  it usually includes,
+            ``pad_size_divisor``, ``pad_value``, ``mean`` and ``std``.
+        init_cfg (dict or list[dict]): Configuration of initialization.
+            Defaults to None.
+    """
+
+    def __init__(self,
+                 detector: Optional[dict] = None,
+                 reid: Optional[dict] = None,
+                 tracker: Optional[dict] = None,
+                 kalman: Optional[dict] = None,
+                 cmc: Optional[dict] = None,
+                 data_preprocessor: OptConfigType = None,
+                 init_cfg: OptConfigType = None):
+        super().__init__(detector, reid, tracker, kalman, data_preprocessor,
+                         init_cfg)
+
+        if detector is not None:
+            self.detector = MODELS.build(detector)
+
+        if reid is not None:
+            self.reid = MODELS.build(reid)
+
+        if kalman is not None:
+            self.kalman = TASK_UTILS.build(kalman)
+
+        if cmc is not None:
+            self.cmc = TASK_UTILS.build(cmc)
+
+        if tracker is not None:
+            self.tracker = MODELS.build(tracker)
```

Comment:
thanks, but `kalman` is needed, because the super() use `self.motion = TASK_UTILS.build(motion)`, but we need `self.kalman = TASK_UTILS.build(kalman)`